get_github_coderabbit_review/coderabbit_fetcher/dashboard/web_server.py

```
@@ -0,0 +1,521 @@
+
+
+@dataclass
+class DashboardConfig:
+    """Dashboard configuration."""
+    host: str = "localhost"
+    port: int = 8080
+    debug: bool = False
+    auto_refresh_interval: int = 5  # seconds
+    max_history_points: int = 100
+    enable_authentication: bool = False
+    secret_key: str = "dashboard-secret-key"
```

Ruff (0.13.1)

Possible hardcoded password assigned to: "secret_key"

(S105)